data/products.json

```diff
@@ -9,41 +9,14 @@
         "link": "https://www.gimp.org/"
       },
       {
-        "name": "Affinity Photo",
```

Comment:
To make the pull request easier to review, let's try to maintain the original order of the products.